css/src/components/checkbox.scss

```diff
@@ -0,0 +1,114 @@
+/* stylelint-disable selector-max-specificity, max-nesting-depth, selector-no-qualifying-type */
+
+$checkbox-border-radius: $radius-sm !default;
+$checkbox-border-width: 0.0625em !default;
+$checkbox-unchecked-border-color: $text !default;
+$checkbox-color: $primary !default;
+$checkbox-check-color: $primary-invert !default;
+$checkbox-check-hover-color: $text-subtle !default;
+$checkbox-size: 1.25em !default;
+$checkbox-check-height: 0.75em !default;
+$checkbox-check-top: 0.1875em !default;
```

Comment:
To my eyes, checkbox top looks better as `.125em;` (more centered vertically). 3px seems to be a bit low

2px (.125em)
![image](https://user-images.githubusercontent.com/30843002/151045158-087478c4-5f1b-47ac-87d6-148933202d4e.png)

3px (.1875em)
![image](https://user-images.githubusercontent.com/30843002/151045195-9f8e02d4-0b60-42cc-8ad2-5e4da7282530.png)

Reply:
Updated